es_ES/_data/catalog/widgets.json

```diff
@@ -0,0 +1,1582 @@
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Table-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "Wrap",
+    "description": "Un widget que muestra sus hijos en múltiples ejecuciones horizontales o verticales.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Multi-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Wrap-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "ListBody",
+    "description": "Un widget que ordena sus hijos secuencialmente a lo largo de un eje dado, forzándolos a la dimensión del elemento primario en el otro eje.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Multi-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/ListBody-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "ListView",
+    "sample": "ListView_index",
+    "description": "Una lista desplazable y lineal de widgets. ListView es el widget de desplazamiento más comúnmente utilizado. Muestra sus hijos uno tras otro en la dirección de desplazamiento. En el eje transversal, los hijos deben completar ListView.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Multi-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/ListView-class.html",
+    "image": "<svg viewBox='0 0 100 100'><filter id='inset-shadow-block' x='-50%' y='-50%' width='200%' height='200%'><feComponentTransfer in='SourceAlpha'><feFuncA type='table' tableValues='1 0' /></feComponentTransfer><feGaussianBlur stdDeviation='2' result='Blur'/><feFlood flood-color='#666666' result='color'/><feComposite in2='Blur' operator='in'/><feComposite in2='SourceAlpha' operator='in' /><feMerge><feMergeNode /></feMerge></filter><rect x='0' y='0' width='100' height='100' fill='#3949ab'/><rect x='30' y='10' width='40' height='80' fill='#ffffff'/><rect x='35' y='10' width='30' height='15' fill='#4dd0e1'/><rect x='35' y='30' width='30' height='20' fill='#4dd0e1'/><rect x='35' y='55' width='30' height='5' fill='#4dd0e1'/><rect x='35' y='65' width='30' height='15' fill='#4dd0e1'/><rect x='35' y='85' width='30' height='5' fill='#4dd0e1'/><rect x='30' y='10' width='40' height='80' fill='#ffffff' filter='url(#inset-shadow-block)'/></svg>"
+  },
+  {
+    "name": "CustomMultiChildLayout",
+    "description": "Un widget que utiliza un delegado para dimensionar y ubicar varios hijos.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Multi-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/CustomMultiChildLayout-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "LayoutBuilder",
+    "description": "Crea un árbol de widgets que puede depender del tamaño del widget principal.",
```

Comment:
can depend on the parent widget's size.
puede depender de los tamaños de los widgets del padre.